Source/Particles/PhotonParticleContainer.cpp

```diff
@@ -72,12 +72,6 @@ PhotonParticleContainer::PushPX(WarpXParIter& pti, Real dt, DtType /*a_dt_type*/
     ParticleReal* const AMREX_RESTRICT ux = attribs[PIdx::ux].dataPtr();
     ParticleReal* const AMREX_RESTRICT uy = attribs[PIdx::uy].dataPtr();
     ParticleReal* const AMREX_RESTRICT uz = attribs[PIdx::uz].dataPtr();
-    const ParticleReal* const AMREX_RESTRICT Ex = attribs[PIdx::Ex].dataPtr();
-    const ParticleReal* const AMREX_RESTRICT Ey = attribs[PIdx::Ey].dataPtr();
-    const ParticleReal* const AMREX_RESTRICT Ez = attribs[PIdx::Ez].dataPtr();
-    const ParticleReal* const AMREX_RESTRICT Bx = attribs[PIdx::Bx].dataPtr();
-    const ParticleReal* const AMREX_RESTRICT By = attribs[PIdx::By].dataPtr();
-    const ParticleReal* const AMREX_RESTRICT Bz = attribs[PIdx::Bz].dataPtr();
```

Comment:
Fixes the warnings:
`./Source/Particles/PhotonParticleContainer.cpp:75:46`: warning: unused variable ‘Ex’ [-Wunused-variable]
     const ParticleReal* const AMREX_RESTRICT Ex = attribs[PIdx::Ex].dataPtr();
`./Source/Particles/PhotonParticleContainer.cpp:76:46`: warning: unused variable ‘Ey’ [-Wunused-variable]
     const ParticleReal* const AMREX_RESTRICT Ey = attribs[PIdx::Ey].dataPtr();
`./Source/Particles/PhotonParticleContainer.cpp:77:46`: warning: unused variable ‘Ez’ [-Wunused-variable]
     const ParticleReal* const AMREX_RESTRICT Ez = attribs[PIdx::Ez].dataPtr();
`./Source/Particles/PhotonParticleContainer.cpp:78:46`: warning: unused variable ‘Bx’ [-Wunused-variable]
     const ParticleReal* const AMREX_RESTRICT Bx = attribs[PIdx::Bx].dataPtr();
`./Source/Particles/PhotonParticleContainer.cpp:79:46`: warning: unused variable ‘By’ [-Wunused-variable]
     const ParticleReal* const AMREX_RESTRICT By = attribs[PIdx::By].dataPtr();
`./Source/Particles/PhotonParticleContainer.cpp:80:46`: warning: unused variable ‘Bz’ [-Wunused-variable]
     const ParticleReal* const AMREX_RESTRICT Bz = attribs[PIdx::Bz].dataPtr();